custom-login/package-lock.json

```diff
@@ -8,6 +8,7 @@
       "version": "7.14.5",
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.14.5.tgz",
       "integrity": "sha512-9pzDqyc6OLDaqe+zbACgFkb6fKMNG6CObKpnYXChRsvYGyEdc7CA2BaqeOM+vOtCS5ndmJicPJhKAwYRI6UfFw==",
+      "dev": true,
```

Comment:
Is it possible to get rid of the lock files in samples folder, so the samples can get the up to date sdk version in a brand new install?  It's removed in the react samples.

Reply:
Done. Removed and added to gitignore